EP.PowerShell.JiraDeployInfo/Public/Update-AzureDeploymentInformation.ps1

```diff
@@ -87,12 +87,12 @@ function Update-AzureDeploymentInformation {
         [Parameter(Mandatory=$false)]
         [ValidateNotNullOrEmpty()]
         [string]
-        $EnvironmentId = $env:ENVIRONMENT_ID,
```

Comment:
The environment name and ID are available from the pipeline vars, so we can use those rather than having to manually set them against each environment with this change.